src/debug.c

```diff
@@ -99,6 +100,9 @@ enum UtilDebugMenu
     DEBUG_UTIL_MENU_ITEM_CHEAT,
     DEBUG_UTIL_MENU_ITEM_EXPANSION_VER,
     DEBUG_UTIL_MENU_ITEM_BERRY_FUNCTIONS,
+    #if RZ_ENABLE == TRUE
+    DEBUG_UTIL_MENU_ITEM_RANDOMIZER,
+    #endif
```

Comment:
```suggestion
#if RZ_ENABLE == TRUE
    DEBUG_UTIL_MENU_ITEM_RANDOMIZER,
#endif
```

This is probably not 100% consistent across the repo, but afaik the general style is to try to deal with #if/#endif indentation like this